joint_trajectory_controller/doc/userdoc.rst

```diff
@@ -140,29 +140,64 @@ open_loop_control (boolean):
 
   If this flag is set, the controller tries to read the values from the command interfaces on starting. If they have real numeric values, those will be used instead of state interfaces. Therefore it is important set command interfaces to NaN (std::numeric_limits<double>::quiet_NaN()) or state values when the hardware is started.
 
-constraints (structure)
+constraints (structure):
```

Comment:
I am not sure what I like more, with or without `:`

You can decide :D

Reply:
you are right, looks nicer without ;)